plugins/scaffolder-react/src/next/components/TaskSteps/StepTime.tsx

```diff
@@ -31,6 +31,16 @@ export const StepTime = (props: {
   const [time, setTime] = useState('');
   const { step } = props;
 
+  const getDelay = () => {
+    if (step.startedAt && step.endedAt && time) {
+      return null;
+    }
+    if (step.startedAt && step.endedAt) {
+      return 1;
```

Comment:
Does this mean that the calculate function will be run a thousand times per second?

Reply:
No, not really :) 

When you open non-running task, it will go to this condition once for each step which was executed and will render time almost immediately, will set `time` variable and then it will go to the first condition what will stop `interval`.

The steps which were not executed yet, will be updated every second.

Reply:
With the implementation in `master` now interval never stops, what's not good